test_add_remove_ldap_role_permission.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+from cassandra import Unauthorized
+
+from longevity_test import LongevityTest
+from sdcm.cluster import BaseNode
+from sdcm.sct_events.system import InfoEvent
+from sdcm.utils.ldap import LdapServerType, LDAP_USERS, LDAP_PASSWORD, LdapUtilsMixin
+
+
+class AddRemoveLdapRolePermissionTest(LongevityTest, LdapUtilsMixin):
+
+    def test_add_remove_ldap_role_permission(self):  # pylint: disable=too-many-statements
+        """
+        Scenario:
+        1. Create new user in Scylla
+        2. Create new role in Ldap containing the new user as a member.
+        3. Verify this new user is unauthorised on resources (create a keyspace).
+        4. Create a new super-user role in Scylla.
+        5. Create a new super-user role in Ldap, with the new user as a member.
+        6. Verify this new user is authorised on resources.
+        7. Modify Ldap role to remove membership of new user.
+        8. Verify the new user is unauthorised on resources (create a table).
+        9. Do clean-up: delete Ldap Role, Scylla roles and keyspace/tables.
+
+        """
+        node: BaseNode = self.db_cluster.nodes[0]
+        stress_queue = []
+        background_write_cmd = "cassandra-stress write no-warmup cl=QUORUM duration=10m -schema 'replication(factor=3)' " \
+                               "-mode cql3 native -rate threads=2 -pop seq=1..1002003 -log interval=5"
+        InfoEvent(message=f"Starting C-S stress load: {background_write_cmd}").publish()
+        stress_queue.append(self.run_stress_thread(stress_cmd=background_write_cmd, round_robin=True))
+
+        if not node.is_enterprise:
+            raise ValueError('Cluster is not enterprise. LDAP is supported only for enterprise. aborting.')
+        if not self.params.get('use_ldap_authorization'):
```

Comment:
shouldn't it work also for `use_ldap_authentication` ??

Reply:
It basically works with authentication, but there's an open issue which blocks this test - https://github.com/scylladb/scylla-enterprise/issues/2641
so better leave it like that.

Reply:
so, if that is the case, i suggest doing the oppositte, and write:

```suggestion
        if self.params.get('use_ldap_authentication'):
            raise ValueError('skipping this test with use_ldap_authentication because of scylla-enterprise#2641')
```

Reply:
@yarongilor , please make this last change

Reply:
fixed

Reply:
i still see the same thing here as before... the if you added afterwards will never work, because of this one... so you should remove this line too